metpy/calc/basic.py

```diff
@@ -300,3 +300,59 @@ def coriolis_parameter(latitude):
 
     """
     return 2. * omega * np.sin(latitude)
+
+
+@exporter.export
+@check_units('[pressure]', '[length]')
+def pressure_at_height_above_pressure(pressure, height):
```

Comment:
Hmm. I could see `add_height_to_pressure`, but to me increment has a very defined value. I'm a bit worried about losing the return type in the name i.e. not ____ at ______. Then again, I suppose we don't need a function for add height to height 😵 

We could even do `height_above_pressure` and just shorten the existing spelling. Thoughts?